Barotrauma/BarotraumaShared/SharedSource/Items/Item.cs

```diff
@@ -1890,42 +1890,54 @@ public bool TryFindController(out Controller controller)
             return controller != null;
         }
 
-        public void SendSignal(int stepsTaken, string signal, string connectionName, Character sender, float power = 0.0f, Item source = null, float signalStrength = 1.0f)
+        public void SendSignal(string signal, string connectionName)
+        {
+            SendSignal(new Signal(signal), connectionName);
+        }
+
+        public void SendSignal(Signal signal, string conenctionName)
```

Comment:
Typo in the parameter name

Reply:
Oops! Fixed in 8b597c8c311232d36b684193ddf9c82752a69e28.